website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -34,7 +37,15 @@ just replace with your customizations and implementations. Following an example.
  
 ### Creating a new project
 
-Let's create the same project but with the Helm plugin:
+Let's check our domain first for we create a new project which will have the same `<group>.<domain>/<version>` API's. We can find our domain value in the CRD's, see:
+
+```yaml
+...
+  group: cache.example.com
+...
+```
+
+Then, let's create a new project with the same domain (`example.com`):
```

Comment:
```suggestion
In Kubebuilder-style projects, CRD groups are defined using two different flags
(`--group` and `--domain).

When we initialize a new project, we need to specify the domain that _all_ APIs in
our project will share, so before creating the new project, we need to determine which
domain we're using for the APIs in our existing project.

To determine the domain, look at the `spec.group` field in your CRDs in the
`deploy/crds` directory.

The domain is everything after the first DNS segment. Using `cache.example.com` as an
example, the `--domain` would be `example.com`.

So let's create a new project with the same domain (`example.com`):
```

The below example is correct based on a domain of `example.com`, so that needs to be fixed.

I think we should show the `init` call separate from `create api`. That way the instructions are the same for all project types:

`````
```sh
$ mkdir nginx-operator
$ cd nginx-operator
$ operator-sdk init --plugins=helm --domain=example.com
```

Now that we have our new project initialized, we need to re-create each of our APIs. 
Using our API example from earlier (`cache.example.com`), we'll use `cache` for the
`--group` flag.

For each API in the existing project, run:
```sh
$ operator-sdk create api \
    --group=cache \
    --api-version=<apiVersion> \
    --kind=<Kind> \
    --helm-chart=<path_to_existing_project>/helm-charts/<chart>
```

`````